nixos/modules/services/home-automation/matter-server.nix

```
@@ -0,0 +1,123 @@
+{ lib
+, pkgs
+, config
+, ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.matter-server;
+in
+
+{
+  meta.maintainers = with lib.maintainers; [ hexa ];
+
+  options.services.matter-server = with types; {
+    enable = mkEnableOption (lib.mdDoc "Matter-server");
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.python-matter-server;
+      defaultText = literalExpression "pkgs.python-matter-server";
+      description = "Package containing the matter-server executable.";
+    };
```

```suggestion
    package = mkPackageOptionMD pkgs "python-matter-server" { };
```

Shorthand for the package option.

---

Thanks for the suggestion! TIL.